server/views/pages/pop/record-checkin-video.njk

```diff
@@ -0,0 +1,56 @@
+{% extends "../../layouts/layout.njk" %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set pageTitle = applicationName + " - Record Video" %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block content %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    {{ govukBackLink({
+      text: "Back to homepage",
+      href: "/pop"
+    }) }}
+  </div>
+</div>
+
+{% if session.checkInVideoPath %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full govuk-!-text-align-center">
+        <h1 class="govuk-heading-l govuk-!-margin-top-1 govuk-!-margin-bottom-3">Uploaded Video</h1>
+      <video src="{{ session.checkInVideoPath }}" controls autoplay class="govuk-!-margin-bottom-3" style="width: 100%;"></video>
+     <div class="govuk-button-group govuk-!-margin-top-3">
+       <a href="/pop/video-checkIn?redo=true" class="govuk-button govuk-button--secondary" style="background-color: white; color: black; border: 1px solid black;">Redo</a>
+       {{ govukButton({
+         text: "Submit checkIn Video",
+         classes: "govuk-button--primary",
+         href: "/pop?checkin=true"
```

Comment:
Even though I bypass sign in to get to the index and go through this process, when I submit and return to the index I think session is being cleared and then I am redirected to one login. I think this might be in error? Testing on Edge

Reply:
This only occurs on local dev set up and works fine in docker